aws-auth-cognito/src/main/java/com/amplifyframework/auth/cognito/util/CognitoAuthExceptionConverter.java

```diff
@@ -32,6 +32,7 @@
 import com.amazonaws.services.cognitoidentityprovider.model.UserNotConfirmedException;
 import com.amazonaws.services.cognitoidentityprovider.model.UserNotFoundException;
 import com.amazonaws.services.cognitoidentityprovider.model.UsernameExistsException;
+import com.amazonaws.services.cognitoidentityprovider.model.MFAMethodNotFoundException;
```

Comment:
This line should be moved to line 29 to fit the import rule.

Reply:
Since you have added `MFAMethodNotFoundException`, I found that there are more Exception types under `com.amazonaws.services.cognitoidentityprovider.model`, such as `EnableSoftwareTokenMFAException`, etc. I'm not sure whether those should be added as well. Could you please reach out to iOS to see whether they have them or not?

Reply:
Yes that's a good idea. These are the exceptions iOS has: https://github.com/aws-amplify/amplify-ios/blob/main/AmplifyPlugins/Auth/AWSCognitoAuthPlugin/Support/Constants/AuthPluginErrorConstants.swift